doc/sphinx/source/tutorials/run-fit.rst

```diff
@@ -56,107 +66,65 @@ without having to tinker with the codebase.
 
-1.  Prepare the fit: ``vp-setupfit runcard.yml``. This command will generate a
+1.  Prepare the fit using ``vp-setupfit``. This command will generate a
     folder with the same name as the runcard (minus the file extension) in the
     current directory, which will contain a copy of the original YAML runcard.
-    The required resources (such as the theory and t0 PDF set) will be
-    downloaded automatically. Alternatively they can be obtained with the
-    ``vp-get`` tool.
+    The required resources will be downloaded, which includes:
+
+      - The t0 PDF set (an LHAPDF object).
+      - The FastKernel tables in the form of a ``theory_xxx.tgz`` file
+      - The postfit evolution operator ``EKO.tar``.
+
+    If the runcard requires to precompute some heavy objects shared among replicas,
+    such as the theory covariance matrix, it will be done during this step.
+
+::
 
-2.  The ``n3fit`` program takes a ``runcard.yml`` as input and a replica number, e.g.
+  vp-setupfit <runcard>.yml
+
+2.  Run the fit using ``n3fit``. The ``n3fit`` program takes a ``runcard.yml`` as input and a replica number, e.g.
     :code:`n3fit runcard.yml replica` where ``replica`` goes from 1-n where n is the
     maximum number of desired replicas. Note that if you desire, for example, a 100
-    replica fit you should launch more than 100 replicas (e.g. 130) because not
-    all of the replicas will pass the checks in ``postfit``
-    (:ref:`see here <postfit>` for more info).
-
-3.  Wait until you have fit results. Then run the ``evolven3fit`` program once to
-    evolve all replicas using DGLAP. The arguments are ``evolven3fit evolve
-    runcard_folder``.
-
-4.  Wait until you have results, then use ``postfit number_of_replicas
-    runcard_folder`` to finalize the PDF set by applying post selection criteria.
-    This will produce a set of ``number_of_replicas + 1`` replicas. This time the
-    number of replicas should be that which you desire in the final fit (100 in the
-    above example). Note that the
-    standard behaviour of ``postfit`` can be modified by using various flags.
+    replica fit you should launch more than 100 replicas (e.g. 120) since not all of them will necessarily converge.
+    While by default the code runs each replica separately, it is possible to run many replicas in parallel, see :ref:`parallel-label`.
+
+::
+
+  for i in {1..120} ; do
+    n3fit <runcard>.yml $i
+  done
+
+3.  Once all replicas have finished, you need to run the ``evolven3fit`` program in order to
+    evolve the PDF from the fitting scale to the whole range of scales needed to create an LHAPDF grid.
+    This is done using the EKO library to perform DGLAP evolution.
+
+::
+
+  evolven3fit evolve <runcard>
+
+4.  Finally, use ``postfit`` to finalize the PDF set by applying post selection criteria and compute the central replica.
+    This will produce a set of ``number_of_replicas + 1`` replicas.
```

Comment:
```suggestion
    This will produce a set of ``number_of_replicas`` error replicas and one mean replica for a total of ``number_of_replicas+1``.
```